internal/services/machinelearning/machine_learning_compute_cluster_resource.go

```diff
@@ -100,6 +100,14 @@ func resourceComputeCluster() *pluginsdk.Resource {
 				ForceNew: true,
 			},
 
+			"node_public_ip_enabled": {
+				Type:         pluginsdk.TypeBool,
+				Optional:     true,
+				Default:      true,
```

Comment:
Hi @mbfrahry ,

Since it defaults to `true` in the swagger: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2023-04-01/machineLearningServices.json#L3534

Is it possible to set `true` into the state when the value isn't returned in the api? Like what I did in this PR: https://github.com/hashicorp/terraform-provider-azurerm/pull/21706/files#diff-0fd2c699ff7d1352db18ad845037d12bfe9df5d528acdbae00977cfd6dc8c579R281

Reply:
Let me try this. It seems like a good idea. Thanks!